config/renovate/renovate.json

```diff
@@ -19,6 +19,9 @@
   "customEnvVariables": {
     "GOTOOLCHAIN": "auto"
   },
+  "vulnerabilityAlerts": {
```

Comment:
iirc, this will enable renovate to check github dependabot alerts, do we know if a repo doesn't have the dependabot alerts enabled or renovate doesn't have permission to check that, will that impact this feature?

Reply:
I tested this in GitLab, so without dependabot, and the CVE data is included.